azure-resources/Network/expressRoutePorts/recommendations.yaml

```diff
@@ -48,3 +48,56 @@
   learnMoreLink:
     - name: Azure Monitor Baseline Alerts - expressRoutePorts
       url: "https://azure.github.io/azure-monitor-baseline-alerts/services/Network/expressRoutePorts/"
+
+- description: Connect on-prem networks to Azure critical workloads via multiple ExpressRoutes peering locations
```

Comment:
I'm confused about this one because it seems to be a duplicate of https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/azure-resources/Network/expressRouteCircuits/#connect-on-prem-networks-to-azure-critical-workloads-via-multiple-expressroutes, when adding the modification above.

Reply:
confirmed with @rodrigosantosms this is per PG request